interfaces/mozillaWot/MozillaWotInterface.js

```diff
@@ -0,0 +1,166 @@
+const fetch = require('node-fetch');
+const WebSocket = require('ws');
+
+const server = require('../../../../libraries/hardwareInterfaces');
```

Comment:
Some other interfaces (e.g. kepware) use the syntax `require('@libraries/hardwareInterfaces');` - not sure if one way is better than the other but pointing it out

Reply:
This one is the eventual standard since electron doesn't play nice with `@libraries`